### DISCUSSION TITLE
what is the difference between curl and rclone?

### DISCUSSION THREAD

**Discussion by ghost** [curl/curl#10609] [Q&A] [answered]
Hi.

I would like to know in terms of project scope what is the difference between curl and rclone. what is the difference between curl and rclone?

**Comment by bagder** [comment]
I had never heard of rclone until this question showed up. It looks like a command line tool for doing transfers to and from cloud storage providers. I can't really provide any comparison as I don't know more than that.

**Comment by abhint** [comment, marked as answer]
I think this is the wrong question. Curl and Rclone have different uses. 

Rclone: `Rclone ("rsync for cloud storage") is a command-line program to sync files and directories to and from different cloud storage providers.`

Curl: `A command line tool and library for transferring data with URL syntax, supporting DICT, FILE, FTP, FTPS, GOPHER, GOPHERS, HTTP, HTTPS, IMAP, IMAPS, LDAP, LDAPS, MQTT, POP3, POP3S, RTMP, RTMPS, RTSP, SCP, SFTP, SMB, SMBS, SMTP, SMTPS, TELNET, TFTP, WS and WSS. libcurl offers a myriad of powerful features`
